src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -61,8 +61,7 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
                 ShowDetailCommandTooltip = Resources.ShowDetailCommandTooltip;
             }
 
-            const ViewerCapabilities tableCaps = (ViewerCapabilities.Table | ViewerCapabilities.List);
-            CanShowOpenCsv = CanShowDetail && result.CanCoerceToDataFrame && _detailsViewer.Capabilities.HasFlag(tableCaps) && result.Length > 0;
+            CanShowOpenCsv = result.CanCoerceToDataFrame && result.Length > 1;
```

Comment:
will do.

---
In reply to: [73923629](https://github.com/Microsoft/RTVS/pull/2157#discussion_r73923629) [](ancestors = 73923629)